libs/cart/testing/src/in-memory-backend/cart.service.integration.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientModule, HttpClient } from '@angular/common/http';
+import { HttpClientInMemoryWebApiModule } from 'angular-in-memory-web-api';
+import { switchMap, tap } from 'rxjs/operators';
+
+import { DaffProductImage } from '@daffodil/product';
+import { DaffProductImageFactory } from '@daffodil/product/testing';
+import { DaffCart } from '@daffodil/cart';
+
+import { DaffInMemoryBackendCartService } from './cart.service';
+import { DaffCartFactory } from '..';
+
+describe('DaffInMemoryBackendCartService | Integration', () => {
+  let cartTestingService: DaffInMemoryBackendCartService;
+  let stubCart: DaffCart;
+  let stubProductImage: DaffProductImage;
+  let daffCartFactory: jasmine.SpyObj<DaffCartFactory>;
+  let daffProductImageFactory: jasmine.SpyObj<DaffProductImageFactory>;
+  let httpClient: HttpClient;
+
+  beforeEach(() => {
+    const daffCartFactorySpy = jasmine.createSpyObj('DaffCartFactory', ['create']);
+    const daffProductImageFactorySpy = jasmine.createSpyObj('DaffProductImageFactory', ['create']);
+
+    stubProductImage = new DaffProductImageFactory().create();
+    stubCart = new DaffCartFactory().create({image: stubProductImage});
+
+    TestBed.configureTestingModule({
+      imports: [
+        HttpClientModule,
+        HttpClientInMemoryWebApiModule.forRoot(DaffInMemoryBackendCartService)
+      ],
+      providers: [
+        { provide: DaffCartFactory, useValue: daffCartFactorySpy},
```

Comment:
Whats the purpose of these spies?

Reply:
They don't seem to have a purpose actually. They were in the spec from before and I didn't touch them. They should be removed.

Reply:
Changed in recent push.